### PR TITLE
separate deploy scripts

### PR DESCRIPTION
**Pull request by saiakilesh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,11 +42,12 @@ Launch a development blockchain node:
 $ yarn hardhat node
 ```
 
-Deploy the contract and write the configuration to a file `nouns.config.json`.
+Deploy the contracts and write the configuration to a file `nouns.config.json`, `zkv.config.json`, `dkg.config.json`.
```
Comment (dtebbs) [inline README.md:45]:
```suggestion
Deploy the contracts and write the configuration to files `nouns.config.json`, `zkv.config.json`, `dkg.config.json`.
```

File: README.md [hunk 2]
```diff
@@ -42,11 +42,12 @@ Launch a development blockchain node:
...
 This file is read by later commands to connect to the contract.
```
Comment (dtebbs) [inline README.md:46]:
```suggestion
These files are read by later commands to connect to the contract.
```

**Review by dtebbs — CHANGES_REQUESTED** [review]
Tiny things in the README, otherwise looks good.

**Comment by saiakilesh** [comment]
Taking care of the merge conflict...give me a min.

**Pull request closed without merging**